packages/terra-clinical-item-collection/tests/wdio/clinical-item-collection-spec.js

```diff
@@ -0,0 +1,99 @@
+/* eslint no-unused-expressions: 0 */
+/* global before, browser, Terra, expect */
+
+Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous').forEach((viewport) => {
+  describe('Item Collection', () => {
+    before(() => {
+      browser.setViewportSize(viewport);
+    });
+
+    describe('Displays an item collection with all possibe elements', () => {
+      before(() => {
+        browser.url('/#/raw/tests/clinical-item-collection/item-collection-all-elements');
+      });
+
+      Terra.should.matchScreenshot(viewport);
+    });
+
+    describe('Displays the correct collection when responding to breakpoint', () => {
+      before(() => {
+        browser.url('/#/raw/tests/clinical-item-collection/item-collection-all-elements');
+      });
+
+      Terra.should.matchScreenshot(viewport);
+    });
+
+    describe('Displays a selectable item collection', () => {
+      before(() => {
+        browser.url('/#/raw/tests/clinical-item-collection/item-collection-selectable');
+      });
+
+      Terra.should.matchScreenshot(viewport);
+    });
+
+    describe('Returns child key for click when onSelect is given to a selectable item collection', () => {
+      before(() => {
+        browser.url('/#/raw/tests/clinical-item-collection/item-collection-selectable');
+        browser.waitForValue('#item1', 3000);
+        browser.click('#item1');
+        browser.click('#selected-key');
+      });
+
+      Terra.should.matchScreenshot(viewport);
+    });
+
+    describe('Returns child key for ENTER when onSelect is given to a selectable item collection', () => {
+      before(() => {
+        browser.url('/#/raw/tests/clinical-item-collection/item-collection-selectable');
+        expect(browser.element('#item2')).to.exist;
+        browser.keys(['\uE004', '\uE004', '\uE004', '\uE007']);
```

Comment:
Nit: for readability, could we change these to 'Tab' and 'Enter' instead of unicode?